tasks/util/make_schema.js

```diff
@@ -12,6 +12,7 @@ module.exports = function makeSchema(plotlyPath, schemaPath) {
         // package is annoying and platform-dependent.
         // see https://github.com/tmpvar/jsdom/issues/1782
         w.HTMLCanvasElement.prototype.getContext = function() { return null; };
+        w.URL.createObjectURL = function() { return null; };
```

Comment:
@etpinard `npm run build` works again with this change.

Reply:
Nicely done!